doc/faq.rst

```diff
@@ -0,0 +1,292 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This document answers common questions about IoTempower. If you don't find your answer here, check our `support page <support.rst>`_ for other ways to get help.
+
+.. contents:: Topics
+   :local:
+   :depth: 2
+
+Getting Started
+===============
+
+What is IoTempower?
+-------------------
+
+IoTempower is a framework for easily creating, deploying, and managing Internet of Things (IoT) systems. It provides tools for configuring wireless sensor/actuator nodes, managing IoT gateways, and integrating with home automation platforms like Home Assistant and Node-RED.
+
+See our `introduction <introduction.rst>`_ for more details.
+
+
+What hardware do I need?
+-------------------------
+
+At minimum, you need:
+
+- **A gateway device**: A Linux computer (like a Raspberry Pi) or a Linux system (native, WSL2, or container)
+- **IoT nodes**: ESP8266 or ESP32 microcontroller boards (like Wemos D1 Mini, NodeMCU, ESP32 DevKit)
+- **Sensors/actuators**: Depending on your project (buttons, LEDs, temperature sensors, relays, etc.)
+
+Check our `hardware guide <hardware.rst>`_ for recommended devices and compatibility information.
+
+
+Which operating systems are supported?
+---------------------------------------
+
+IoTempower works on:
+
+- **Linux** (Ubuntu, Debian, Arch, Manjaro) - recommended, best experience
+- **macOS** (with Homebrew)
+- **Windows** via WSL2 (Windows Subsystem for Linux)
+- **Android** via Termux
+- **Containerized** via Podman (Docker is deprecated)
+
+See our `installation guide <installation.rst>`_ for platform-specific instructions.
+
+
+Installation & Setup
+====================
+
+How do I install IoTempower?
+-----------------------------
+
+The quickest way is using the installation script:
+
+.. code-block:: bash
+
+   curl -L https://now.iotempower.us | bash -s -- --default
+
+For detailed instructions and alternative methods, see our `installation guide <installation.rst>`_.
+
+
+Where is IoTempower installed by default?
+------------------------------------------
+
+The default installation directory is ``/iot``. You can customize this during installation if needed.
+
+
+How do I update IoTempower?
+----------------------------
+
+Run the following command from within your IoTempower installation:
+
+.. code-block:: bash
+
+   iot upgrade
+
+This will pull the latest changes and update your installation.
+
+
+Device Configuration
+====================
+
+How do I add a new device/sensor?
+----------------------------------
+
+To add a device to a node:
+
+1. Navigate to your node directory (e.g., ``/iot/projects/myproject/mynode``)
+2. Edit the ``setup.cpp`` file
+3. Add device initialization code using IoTempower's device classes
```

Comment:
.. you can usually do that in the lib/node_help/esp folder if it's a generic device (after compile, you can use the created build directory as a Platformio project and debug and edit your files there as they are "just" symbolic links, make use of ulog for debugging!)